src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,29 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+    public void loginCorrectCred() {
+        provideEmail("dagebeyehu@gmail.com");
+        providePassword("yozca5uQiLxb!");
+        clickSubmit();
+    }
+
+
+    public void provideEmail(String email) {
+        enterText(By.cssSelector("input[type='email']"),email);
+    }
+
+    public void providePassword(String password) {
+        enterText(By.cssSelector("input[type='password']"),password);
+    }
+
+    public void clickSubmit() {
+        clickOnElement(By.cssSelector("[type='submit']"));
+    }
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)